tests/texture.py

```diff
@@ -338,6 +338,72 @@ def texture_3d(cfg: SceneCfg):
     return render
 
 
+_STEPS = 4
+
+
```

Comment:
you can stitch the constants together, the double line break is for functions

Reply:
Removed the constant.